boot/freeldr/freeldr/arch/i386/multiboot.S

```diff
@@ -179,17 +176,14 @@ mbfail:
     int 3
 mbstop:
     jmp short mbstop  /* We should never get here */
-
-.code16
 mb4:
     /* Disable protected mode */
-    mov eax, cr0
-    and eax, CR0_PE_CLR
-    mov cr0, eax
+    mov rax, cr0
+    data32 and eax, CR0_PE_CLR
+    mov cr0, rax
 
     /* Jump to real entry point */
-    ljmp16 0, FREELDR_BASE
-.endcode16
+    ljmp16 HEX(00), FREELDR_BASE
```

Comment:
The "0" -> "HEX(00)" change doesn't seem necessary

Reply:
Reverted!